src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,28 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+
+        <p>
+          {' Posted by  '}
+          <UserInfo user={post.user[0]} />
```

Comment:
```suggestion
          {post.user && <UserInfo user={post.user} />}
```
find can return undefined, so better to add the check